components/alibi-detect-server/Dockerfile

```diff
@@ -14,28 +14,38 @@ FROM base as builder
 ARG PYTHON_VERSION
 ARG CONDA_VERSION
 
+# mesa-libGL: this is to avoid "ImportError: libGL.so.1" from opencv
+RUN microdnf update -y && \
+    microdnf install -y \
+      unzip mesa-libGL \
+      make automake gcc gcc-c++
+
 # Install Rclone Binary to be present in the image
-RUN yum install -y unzip
 RUN wget https://downloads.rclone.org/v1.55.1/rclone-v1.55.1-linux-amd64.zip && \
     unzip rclone-v1.55.1-linux-amd64.zip && \
     mv rclone-v1.55.1-linux-amd64/rclone /usr/bin/rclone && \
     rm -rf rclone-v1.55.1-linux-amd64.zip rclone-v1.55.1-linux-amd64
 
-
 # Install Python / Conda
-RUN conda install --yes python=${PYTHON_VERSION} conda=${CONDA_VERSION}
-RUN pip install pip==21.2.4 setuptools==58.1.0
-RUN dnf install -y make automake gcc gcc-c++
+# Note that we need to force Conda to use the system's std-c++ library, as
+# otherwise PyStan won't compile with Conda's older stdc++ library:
+# https://github.com/stan-dev/pystan/issues/294#issuecomment-870711100
+RUN conda install --yes -c conda-forge \
```

Comment:
So the new thing is opting for `conda-forge` channel here - I wonder what is the exact reasoning for it? Could it be that this is what relates to the need for linking `libstdc`?

Reply:
I think I tried both, and neither worked. TBH this is mainly an artefact of trying out diff things to see what worked. 

Happy to revert it back, although I'd be keen to start using `conda-forge` more across our images - mainly to workaround Conda's licensing constrains (haven't checked what's the latest on those though).

Reply:
Fair point, I'd just to just include it in all images for consistency